cmake/ConfigureCompiler.cmake

```diff
@@ -161,16 +168,19 @@ else()
     -mmmx
     -mavx
     -msse4.2)
-  add_compile_options($<$<COMPILE_LANGUAGE:CXX>:-std=c++17>)
+
+  set(CMAKE_CXX_STANDARD 17)
```

Comment:
Does this work on Windows too? If so I would suggest you change the Windows compile options as well

Reply:
agreed